### DISCUSSION TITLE
Does  assertj-assertions-generator and assertj-assertions-generator-maven-plugin still lived ?

### DISCUSSION THREAD

**Discussion by SamSuffit** [assertj/assertj#2987] [General]
Hello, 

I’ve got a question. 
I used assertj-assertions-generator-maven-plugin for years. But with javax being deprecated and been replace by jakarta I found it difficult to use. 
I can used custom template but all developement  seams to be already done and only a reulase of  both projet is missing. 

Does anyone know if it will append soon ?

**Comment by joel-costigliola** [comment]
I still like the idea of the generator but unfortunately have no time for it these days.
It would be great if someone could step up and give another life to these projects

**Reply by SamSuffit** [reply]
I understand. 
I check with my teams and unfortunatly we won’t be able to step up.
